github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml

```diff
@@ -29,9 +31,10 @@ postsubmits:
         - "-c"
         - |
           cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io &&
-          # Only push images on tags
+          # Only push images on tags, for amd64 and aarch64 (crossbuild, since we have no aarch64 builder)
           [ -z "$(git tag --points-at HEAD | head -1)" ] ||
-          DOCKER_TAG="$(git tag --points-at HEAD | head -1)" make bazel-push-images
+          DOCKER_TAG="$(git tag --points-at HEAD | head -1)-amd64" make bazel-push-images ||
```

Comment:
Reassigning `DOCKER_TAG`? Is this what you meant?

Reply:
yes if you look at what kubevirt does here, they do the same, making a tag for amd and arm and quay seems to correctly re-arrange that.